src/hockeypuck/pghkp/storage.go

```diff
@@ -67,8 +67,13 @@ keywords tsvector
 rfingerprint TEXT NOT NULL,
 rsubfp TEXT NOT NULL PRIMARY KEY,
 FOREIGN KEY (rfingerprint) REFERENCES keys(rfingerprint)
-)
-`,
+)`,
+	`CREATE TABLE IF NOT EXISTS blacklist (
```

Comment:
Suggestion: `s/blacklist/blocked_rfingerprint/g` to better convey what it does.

Reply:
Yeah, the 'blacklist' terminology needs to be changed elsewhere also. I'd also prefer not to use _R_fingerprint here if I can help it, but that depends on #285 first...